taxonomy-editor-frontend/src/pages/editentry/AccumulateAllComponents.jsx

```diff
@@ -27,7 +28,23 @@ const AccumulateAllComponents = ({ id }) => {
 
     // Setting state of node after fetch
     useEffect(() => {
-        setNodeObject(node?.[0]);
+        if (node) {
```

Comment:
should it do something if `!not` ? like setting to `null` or something?

Reply:
I don't think it's necessary since the initial state is already null. Only when the node info is populated, we need to update the state.

Reply:
what about if url changes and data is renewed?